clients/src/main/resources/common/message/VoteRequest.json

```diff
@@ -18,7 +18,8 @@
   "type": "request",
   "listeners": ["controller"],
   "name": "VoteRequest",
-  "validVersions": "0",
+  // Version 1 adds the PreVote field and renames CandidateEpoch and CandidateId to ReplicaEpoch and ReplicaId
```

Comment:
Should we set `latestVersionUnstable` until the feature is fully implemented?

Reply:
@hachikuji I think we can avoid this if we implement the handling of version 1 requests in this PR. See https://github.com/apache/kafka/pull/15231#discussion_r1463743747

Reply:
Agree it would be better to implement all of it here.